### ISSUE TITLE
[Feat] Support min opacity value for heatmaps

### ISSUE THREAD

**Issue by bobrik** [grafana/grafana#11838]
Carrying over from https://github.com/grafana/grafana/issues/8539#issuecomment-385704636. cc @alexanderzobnin.

Currently it's impossible to see outliers in the light theme:

![image](https://user-images.githubusercontent.com/89186/39403202-bddc4bca-4b29-11e8-9406-cc979e452bf4.png)

Dark theme is slightly better:

![image](https://user-images.githubusercontent.com/89186/39403205-d1e2d7c4-4b29-11e8-954b-1a347e699cf4.png)

Compare to this, where every outlier is clearly visible:

![image](https://user-images.githubusercontent.com/89186/39403206-e2db8166-4b29-11e8-93ac-ca10ed6f3896.png)

Switching to spectrum mode does not help:

* Opacity mode:

![image](https://user-images.githubusercontent.com/89186/39479991-c0e03d4c-4d1b-11e8-8a09-91a4dc55ab8e.png)

* Spectrum mode:

![image](https://user-images.githubusercontent.com/89186/39480025-d50e8e4a-4d1b-11e8-9153-88a302970040.png)

In summary:

* For opacity we want min opacity value to see outliers even with minimal value
* For spectrum mode we want transparent cells for events that never happened
